bin/deferred.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "../lib/tom_queue.rb"
```

Comment:
This file is just for showcasing. We should delete it and replace with a real script to launch the process